harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/geoPREST/Harvester.java

```diff
@@ -262,6 +265,33 @@ private RecordInfo getRecordInfo(Element record) {
 
     }
 
+    protected Date parseDateJdk11(String pubDate) throws ParseException {
+        Locale wellKnownLocales[] = {Locale.ENGLISH, Locale.FRENCH, Locale.GERMAN, Locale.ITALIAN};
+
+        for (Locale locale : wellKnownLocales) {
+            DateTimeFormatter formatter = DateTimeFormatter.RFC_1123_DATE_TIME.withLocale(locale);
+
+            try {
+                ZonedDateTime date = ZonedDateTime.parse(pubDate, formatter);
+                return Date.from(date.toInstant());
+            } catch (DateTimeParseException e) {
+                // workaround for https://bugs.openjdk.java.net/browse/JDK-8136539
+                if(locale == Locale.GERMAN && (pubDate.toLowerCase(Locale.GERMAN).contains("mrz")
+                || pubDate.toLowerCase(Locale.GERMAN).contains("mär"))) {
+                    try {
+                        log.info("Applying MRZ workaround to '"+pubDate+"'");
+                        String wad = pubDate.toLowerCase(Locale.GERMAN).replace("mrz", "mar");
+                        wad = wad.replace("mär", "mar");
+                        ZonedDateTime workedAroundDate = ZonedDateTime.parse(wad, formatter);
+                        return Date.from(workedAroundDate.toInstant());
+                    } catch (DateTimeParseException ex) {}
+                }
+                log.debug("Date '"+pubDate+"' is not parsable according to " + locale);
```

Comment:
Apply formatting to the code:
```suggestion
                log.debug("Date '" + pubDate + "' is not parsable according to " + locale);
```